libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.java

```diff
@@ -473,8 +469,8 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
                 }
                 // Or succeed?
                 else {
-                    TokenEndpointResponse tr = new TokenEndpointResponse(params);
-                    onAuthFlowComplete(tr);
+                    String code = params.get("code");
```

Comment:
When we intercept the redirect to the callback uri, we should have a code in the params.